docs/node/server-plugins.md

```diff
@@ -40,6 +40,12 @@ ones:
   context values are passed to the service implementation. See
   [Context values](./interceptors.md#context-values) for more information.
 
+:::note
+Over HTTP/2, Node.js can serve the Connect, gRPC, and gRPC-Web protocols with
+all variants of RPCs.
+Over HTTP 1.1, the gRPC protocol and bidirectional streaming RPCs are not
+supported.
```

Comment:
This isn't entirely accurate: gRPC works fine over HTTP 1.1 (except bidi). Our implementation supports it, but most others don't. It's very likely that users run into issues with other implementations over H1.